core/node/events/snapshot.go

```diff
@@ -144,7 +144,10 @@ func make_SnapshotMembers(iInception IsInceptionPayload, creatorAddress []byte)
 			}, &MemberPayload_Snapshot_Member{
```

Comment:
can we fix this to make the MemberPayload_Snapshot once and then do the custom joined logic in the switch statement

Reply:
pushed an update